### PR TITLE
feat(device types): added user facing identifier for devices

### PR DESCRIPTION
**Pull request by KZavorotna**
Is required for https://github.com/withthegrid/platform-client/pull/1545

### CODE REVIEW

File: src/routes/device/update-multiple-devices.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import Joi from 'joi';
```
Comment (everhardt) [inline src/routes/device/update-multiple-devices.ts:1]: * Shall we apply the same route naming as we use in https://github.com/withthegrid/platform-sdk/pull/287? So `updateDevices`?
* Taking https://github.com/withthegrid/platform-sdk/pull/287/files#r968069994 into account, shall we rename the route `path` to `/devices`?

**Review by everhardt — CHANGES_REQUESTED** [review]
Looks good, 1 request.

File: src/routes/device/update-multiple-devices.ts (hunk 1 above)
Comment (KZavorotna) [reply src/routes/device/update-multiple-devices.ts]: Updated

**Review by everhardt — APPROVED** [review]
👍

**Review by Arinono — APPROVED** [review]
I didn't find anything out of the ordinary. Looking good! 👌

**Comment by srieding** [comment]
No point in having a 4th pair of eyes look at this I would say

**Pull request merged**